lib/src/main/java/io/fabianterhorst/isometric/Isometric.java

```diff
@@ -123,6 +145,18 @@ public void measure(int width, int height, boolean sort) {
                 item.transformedPoints[i] = translatePoint(point);
             }
 
+            //remove item if not in view
+            //the if conditions here are ordered carefully to save computation, fail fast approach
+            if ((cull && cullPath(item)) || !this.itemInDrawingBounds(item)) {
+                //the path is invisible. It does not need to be considered any more
+                this.items.remove(itemIndex);
```

Comment:
In one of my earlier commits, i marked each item with a boolean on whether it should draw or not, when i did this, the items still went through the sorting calculations. By removing the items from the collection they are no longer considered in the sorting or drawing calcs. Plus, it is one less variable to have to maintain. What are your thoughts?

Reply:
Best way to not break compatibility would be to add another option to make the itemInDrawingBounds check optional as well. Before the 1.0 release im going to find an more modular way since each application will probably have different usages.